crates/katana/executor/Cargo.toml

```diff
@@ -10,14 +10,12 @@ version.workspace = true
 katana-primitives.workspace = true
 katana-provider.workspace = true
 
-convert_case.workspace = true
 parking_lot.workspace = true
 starknet = { workspace = true, optional = true }
 thiserror.workspace = true
 tracing.workspace = true
 
-# blockifier deps
-blockifier = { git = "https://github.com/dojoengine/blockifier", rev = "57c11586", features = [ "testing" ], optional = true }
+blockifier = { git = "https://github.com/dojoengine/blockifier", rev = "24029ba4", features = [ "testing" ], optional = true }
```

Comment:
If we are on a very specific upstream tag, I would say it could be great to have the same tag with `dojo` prefix/suffix to make it more evident for the matching versions.
But in this case it seems to be a precise commit more than a tag though. So it may not apply. :)

Reply:
yeah good point will do that

Reply:
on second thought, i think i'll just point it to the branch name (`cairo-2.7`) instead of creating a specific tag.